pkg/controller/installation/core_controller.go

```diff
@@ -241,12 +241,12 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 			return fmt.Errorf("tigera-installation-controller failed to watch Secret %s: %w", render.ElasticsearchAdminUserSecret, err)
 		}
 
-		if err = utils.AddSecretsWatch(c, relasticsearch.PublicCertSecret, rmeta.OperatorNamespace()); err != nil {
-			return fmt.Errorf("tigera-installation-controller failed to watch Secret '%s' in '%s' namespace: %w", relasticsearch.PublicCertSecret, rmeta.OperatorNamespace(), err)
+		if err = utils.AddSecretsWatch(c, relasticsearch.InternalCertSecret, rmeta.OperatorNamespace()); err != nil {
```

Comment:
You've changed these watches but in the Reconcile you're still getting the Public version of this and the Kibana CertSecrets.

Reply:
Ah right that was likely a find + replace mistake, I'll revert.